### ISSUE TITLE
Add tests for ClientHandler

### ISSUE THREAD

**Issue by Vimbayinashe** [fungover/storm#42]
ClientHandler currently has a code coverage of 0% in the existing tests.

**Comment by LordRekishi** [comment]
It might be difficult to make functional tests for the ClientHandler and Server since their functionality is tied to connecting clients and sending questions and receiving responses through the network.

**Comment by Vimbayinashe** [comment]
Yes, I agree :sweat_smile:! Martin mentioned that it is possible to mock a socket byte stream as an argument for ClientHandler's constructor and test its _run_ method when I was working on issue #11 so I want to give it a try.

**Comment by LordRekishi** [comment]
> Yes, I agree 😅! Martin mentioned that it is possible to mock a socket byte stream as an argument for ClientHandler's constructor and test its _run_ method when I was working on issue #11 so I want to give it a try.

Go for it! You might also be able to use my Client stub class found in the test folder, together with what I did in @BeforeAll in my ServerTest
